platform/core/src/core.rs

```diff
@@ -1,13 +1,35 @@
 use crate::{core_failure::CoreFailure, feature_categories, io::file_system::DefaultFileSystem};
 use once_cell::sync::OnceCell;
+use serde::{Deserialize, Serialize};
+
+#[cfg(target_os = "android")]
+use android_logger::Config;
+#[cfg(target_os = "android")]
+use log::{trace, Level};
 
 // Core is held as a singleton.
 static CORE: OnceCell<Core> = OnceCell::new();
 
+#[derive(Debug, Clone, Serialize)]
+pub enum NativeRequest {
+    Example,
+}
+
+#[derive(Debug, Clone, Serialize, Deserialize)]
+pub enum NativeResponse {
+    Example(String),
+}
```

Comment:
Nit: Probably these can be to be moved to a separate file, as the content will grow.